js/controller.js

```diff
@@ -1445,7 +1445,11 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
           }
         }
       } else {
-        remainingTime = parseInt(this.state.adVideoDuration - (this.state.adVideoPlayhead * 100));
+        if (isSSAI) {
+          remainingTime = parseInt(this.state.adVideoDuration - (this.state.adVideoPlayhead * 100));
```

Comment:
What units are adVideoPlayhead in for SSAI? Is it possible to have the adVideoPlayhead be the same units for both SSAI and non-SSAI? This will help maintenance down the road.

Reply:
Actually I just checked that SSAI is always "Live" even for VOD, so those assets will never enter in this else condition. I will remove this if condition.